cmd/root.go

```diff
@@ -161,6 +161,20 @@ func init() {
 		"Set a request timeout for the client to connect with Kong (in seconds).")
 	viper.BindPFlag("timeout",
 		rootCmd.PersistentFlags().Lookup("timeout"))
+
+	rootCmd.PersistentFlags().String("tls-client-cert", "",
```

Comment:
Can you please rename this to `tls-client-cert-file`. That's more in line with what we do somewhere else in this repo.

Reply:
The expected content is a string representing the certificate, so would you rather think about passing a file instead ?
I will renamed then.

Reply:
Oh, I misread it then, apologies. Would you mind supporting both ways then?

Reply:
Sure will do.